### ISSUE TITLE
[BUG] Entity sensor.home_cloud_ceiling (<class 'custom_components.weatherdotcom.sensor.WeatherSensor'>) is using native unit of measurement '' which is not a valid unit for the device class ('distance') it is using

### ISSUE THREAD

**Issue by Scoop2389** [jaydeethree/Home-Assistant-weatherdotcom#16]
When starting home assistant, the following message is logged:
```
2023-10-31 16:50:04.489 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.home_cloud_ceiling (<class 'custom_components.weatherdotcom.sensor.WeatherSensor'>) is using native unit of measurement '' which is not a valid unit for the device class ('distance') it is using; expected one of ['yd', 'km', 'in', 'cm', 'mm', 'm', 'mi', 'ft']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/jaydeethree/Home-Assistant-weatherdotcom/issues/
```

**Comment by jaydeethree** [comment]
This should have been fixed yesterday with version 1.1.2 - please upgrade to it and let me know if you continue to see problems.

**Comment by Scoop2389** [comment]
Thanks, HACS didn't show an update was available until I manually checked.
